tests/run-make/atomic-lock-free/rmake.rs

```diff
@@ -0,0 +1,52 @@
+// This tests ensure that atomic types are never lowered into runtime library calls that are not
+// guaranteed to be lock-free.
+
+use run_make_support::{is_linux, llvm_components_contain, llvm_readobj, rustc};
+
+fn compile(target: &str) {
+    rustc().input("atomic_lock_free.rs").target(target).run();
+}
+
+fn check() {
+    llvm_readobj()
+        .symbols()
+        .input("libatomic_lock_free.rlib")
+        .run()
+        .assert_stdout_not_contains("__atomic_fetch_add");
+}
+
+fn compile_and_check(target: &str) {
+    compile(target);
+    check();
+}
+
+fn main() {
+    if is_linux() {
```

Comment:
does rmake support the `//@ only-linux` pragmas?

Reply:
Yes, it should

Reply:
Suggestion: Yeah please use `//@ only-linux` directive over the `is_linux` check here so the test is properly registered as "ignored" when running on non-linux instead of silently passing.